content/400-reference/200-api-reference/100-prisma-schema-reference.mdx

```diff
@@ -980,7 +980,7 @@ Attributes modify the behavior of a [field](#model-fields) or block (e.g. [model
 - _Field_ attributes are prefixed with `@`
 - _Block_ attributes are prefixed with `@@`
 
-Some attributes take arguments. Arguments in attributes are always named, but in most cases the argument _name_ can be omitted.
+Some attributes take arguments. Arguments in attributes are always named, but in some cases one of the argument's _name_ can be omitted.
```

Comment:
I think it can only ever be the first one that can omit the name.

Reply:
Is the order important? 
Or what are you suggesting to change here?